.github/bin/build-binaries-matrix

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env ruby
+require "json"
+
+RELEASE_NAME = "Binaries"
+
+EXISTING_ASSETS = JSON.parse(`
+  gh --repo freckle/weeder-action release view #{RELEASE_NAME} --json assets |
+    jq '.assets | map(.name)'
+`)
+
+GHCs = %W[
+  9.2.5
+  9.2.4
+  9.2.3
+  9.2.2
+  9.0.2
+  9.0.1
+  8.10.7
+  8.10.6
+  8.10.5
+  8.10.4
+  8.10.3
+  8.10.2
+  8.10.1
+  8.8.4
+  8.8.3
+  8.8.2
+  8.8.1
+]
```

Comment:
Side quest, would it make sense to contribute this matrix to `weeder` itself and pull binaries from that project?

https://github.com/ocharles/weeder/releases

Reply:
@ocharles is using `nix`, which makes binary caching easy, but I'm not sure the current state of multi GHC builds in that ecosystem.

Reply:
I was indeed surprised when I found they weren't there. It's possible needing to have a binary built against the GHC of your project was enough complexity that they punted -- or maybe there's an Actual Reason they'd not want it. I'll probably open an Issue to ask at some point.